test/lib/specHelperSpec.js

```diff
@@ -108,5 +115,19 @@ describe('Fluxxed up test helpers', function() {
       // Signal that test should be finished:
       rig.finish()
     })
+
+    it('toggles the radio button', () => {
+      var element = rig.domNode.find('.radio-input')[0]
+      rig.toggleRadioButton(element)
```

Comment:
@andela-tkomolafe Same here: this can become `rig.toggleRadioButton('.radio-input')`